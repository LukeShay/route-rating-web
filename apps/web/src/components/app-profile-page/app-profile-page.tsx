import { Component, Host, h } from '@stencil/core'

@Component({
  tag: 'app-profile-page',
  styleUrl: 'app-profile-page.scss',
  shadow: true,
})
export class AppProfilePage {
  render() {
    return (
      <Host>
        <h1>Profile</h1>
      </Host>
    )
  }
}
