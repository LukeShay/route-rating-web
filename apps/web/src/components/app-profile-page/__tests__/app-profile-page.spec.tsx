import { AppProfilePage } from '../app-profile-page'
import { newSpecPage } from '@stencil/core/testing'

describe('app-profile-page', () => {
  it('should match snapshot', async () => {
    const page = await newSpecPage({
      components: [AppProfilePage],
      html: `<app-profile-page></app-profile-page>`,
    })
    expect(page.root).toMatchSnapshot()
  })
})
