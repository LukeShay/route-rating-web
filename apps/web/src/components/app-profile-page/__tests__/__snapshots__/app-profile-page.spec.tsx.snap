// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-profile-page should match snapshot 1`] = `
<app-profile-page>
  <mock:shadow-root>
    <h1>
      Profile
    </h1>
  </mock:shadow-root>
</app-profile-page>
`;
